src/components/CampaignTextersForm.jsx

```diff
@@ -274,6 +275,7 @@ export default class CampaignTextersForm extends React.Component {
 
     const dataSource = orgTexters
       .filter(orgTexter => !texters.find(texter => texter.id === orgTexter.id))
+      .filter(orgTexter => getHighestRole(orgTexter.roles) === "SUSPENDED")
```

Comment:
I think this needs to be `!==` though

Reply:
Good eye! I was testing this yesterday and was confused why it wasn't working. Went looking elsewhere and decided to come back to it today. ✅  problem solved 😄 
Fixed in https://github.com/MoveOnOrg/Spoke/pull/1633/commits/2eeb0c9ecc11b634a20eaea35a92fcd960c860e2